include/onnxruntime/core/framework/func_api.h

```diff
@@ -21,7 +21,7 @@ using FunctionState = void*;
 // take the ComputeContext, and create a function state.
 using CreateFunctionStateC = int (*)(ComputeContext*, FunctionState*);
 // pass in the function state and input/output tensors, perform compute and return status
-using ComputeFuncC = Status (*)(FunctionState, const OrtApi*, OrtKernelContext*);
+using ComputeFuncC = common::Status (*)(FunctionState, const OrtApi*, OrtKernelContext*);
```

Comment:
>common::Status [](start = 21, length = 14)

Can you make it a fully qualified name so using directive above is not needed?

Reply:
Aha, I wanted to do that! I put a comment on this line above:

`using onnxruntime::common::Status;  // TODO: Needed by WinML, but shouldn't be put into the global namespace like this
`

So I originally removed the 'using' at the top but since WinML uses it without any scoping I'd need to change to their code to fix. Since that started to go a bit too far out of scope I put the using back and left a comment (but I left my common::Status changes since that's how it should be). Alas.